Cache envs refactoring (#394)
blockscout/docs: setup/env-variables/backend-env-variables.md

| ----------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------- |
| `CACHE_TXS_COUNT_PERIOD`                              | Time interval to restart the task that calculates the total txs count. Starting from release v6.8.0+, if the value is not set, ttl value gradually increases until the default value with growth of the block numbers.                                                                                                                     | <p>Version: v4.1.3+<br>Default: <code>2h</code><br>Applications: API, Indexer</p>                           |

> | `CACHE_ADDRESS_COUNT_PERIOD`         | Interval to restart the task which calculates number of all addresses.                                                                                                                                                                                                                                                                     | <p>Version: master<br>Default: <code>30m</code><br>Applications: API, Indexer</p>   |

Reviewer: ```suggestion
| `CACHE_ADDRESS_COUNT_PERIOD`         | Interval for restarting the task that calculates the total number of addresses.                                                                                                                                                                                                                                                                     | <p>Version: master<br>Default: <code>30m</code><br>Applications: API, Indexer</p>   |
```

| `CACHE_ADDRESS_TRANSACTIONS_COUNTER_PERIOD`           | Time to live of address' transaction counter. [Time format](backend-env-variables.md#time-format). Introduced in [#3330](https://github.com/poanetwork/blockscout/pull/3330).                                                                                                                                                              | <p>Version: v3.4.0+<br>Default: <code>1h</code><br>Applications: API, Indexer</p>                           |
| `CACHE_ADDRESS_TOKENS_USD_SUM_PERIOD`                 | Managing of cache invalidation period for the sum of USD value of tokens per tokens' holder address.                                                                                                                                                                                                                                       | <p>Version: v3.5.0+<br>Default: <code>1h</code><br>Applications: API, Indexer</p>                           |

> | `CACHE_EXCHANGE_RATES_PERIOD`                         | Value to tune the time to live of exchange rates. [Time format](backend-env-variables.md#time-format). Implemented in [#5671](https://github.com/blockscout/blockscout/pull/5671).                                                                                                                                                         | <p>Version: v4.1.5+<br>Default: <code>10m</code><br>Applications: API, Indexer</p>   |

Reviewer: Just to get common understanding, this change is not related to current backend PR and is just a general fix, right? If so, close the thread

Author: Yes, `CACHE_EXCHANGE_RATES_PERIOD` converted to runtime var a while ago. Just adjusting docs to that change.
